apps/test/unit/templates/instructions/InlineAudioTest.js

```diff
@@ -104,16 +113,43 @@ describe('InlineAudio', function() {
     expect(component.exists('.inline-audio')).to.be.true;
   });
 
-  it('can toggle audio', function() {
+  it('can toggle audio', async function() {
     const component = mount(<StatelessInlineAudio {...DEFAULT_PROPS} />);
 
     expect(component.state().playing).to.be.false;
     component.instance().toggleAudio();
+    await waitForPromises();
     expect(component.state().playing).to.be.true;
     component.instance().toggleAudio();
+    await waitForPromises();
     expect(component.state().playing).to.be.false;
   });
 
+  it('autoplays if autoplay of text-to-speech is enabled', async function() {
+    const component = mount(
+      <StatelessInlineAudio
+        assetUrl={function() {}}
+        ttsAutoplayEnabled={true}
+      />
+    );
+
+    await waitForPromises();
+    expect(component.state().playing).to.be.true;
+  });
+
+  it('when playAudio resolves, state.playing set to true', async () => {
+    const component = mount(
+      <StatelessInlineAudio
+        assetUrl={function() {}}
+        ttsAutoplayEnabled={false}
+      />
+    );
+
+    expect(component.state().playing).to.be.false;
+    await component.instance().playAudio();
+    expect(component.state().playing).to.be.true;
+  });
+
```

Comment:
I attempted for 2 days to figure out how to stub the DOMException and check that playing audio starts up after the first user interaction but was getting a lot of trouble with the promises involved in that code path and tests not waiting on promises. I involved other engineers and ultimately decided it was not worth the time it was taking. Please forgive this slight lack of coverage.